src/ImageGallery.js

```diff
@@ -237,12 +237,15 @@ class ImageGallery extends React.Component {
 
   onThumbnailClick(event, index) {
     const { onThumbnailClick } = this.props;
+    const { currentIndex } = this.state;
     // blur element to remove outline cause by focus
     event.target.parentNode.parentNode.blur();
+   if(currentIndex!==index){
```

Comment:
i'm not sure why eslint isn't picking this up for you but this should be formatted like
`if (currentIndex !== index) {`

Reply:
Not sure. Prettier as formatter works for me but it changes entire file.